src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,24 +14,30 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
 
     private final UniqueTagList tags;
+    private static int nextSequenceNumber = 0;
+    
     /**
      * Assumption: Every field must be present and not null.
+     * @param  
      */
     public Person(Name name, Phone phone, Email email, Address address, UniqueTagList tags) {
         this.name = name;
         this.phone = phone;
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = ++nextSequenceNumber;
```

Comment:
... then here, it would be more meaningful to assign sequence number with the `nextSequenceNumber`, then increment.